static/app/components/banner.tsx

```diff
@@ -32,21 +32,15 @@ type BannerWrapperProps = {
 };
 
 type Props = BannerWrapperProps & {
+  children?: React.ReactNode;
```

Comment:
but not required here

Reply:
In the test we render it without any children